app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
+
```

Comment:
- [ ] As per the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the column name should be `author_id`. You can achieve this by writing like this.

Hint: 
```
has_many :comments, foreign_key: :author_id
  has_many :posts, foreign_key: :author_id
  has_many :likes, foreign_key: :author_id
```

and in the respective file `post.rb`, `comment.rb`, and `like.rb` you have to write like this

`belongs_to :author, class_name: 'User'` instead of `belongs_to :user`.

Reply:
- [ ] Please add those required changes requested by my colleague as he stated (hints).